cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaClusterTest.java

```diff
@@ -84,10 +91,21 @@ private void checkHeadlessService(Service headless) {
     public void testGenerateStatefulSet() {
         // We expect a single statefulSet ...
         StatefulSet ss = kc.generateStatefulSet(true);
-        checkStatefulSet(ss);
+        checkStatefulSet(ss, cm);
     }
 
-    private void checkStatefulSet(StatefulSet ss) {
+    @Test
+    public void testGenerateStatefulSetWithRack() {
```

Comment:
Do we have any test where we have both init containers? I think we should have one.